lib/Test/Nginx/Util.pm

```diff
@@ -2337,6 +2344,9 @@ retry:
             #warn "pid file not found";
         }
     }
+
+    return if !($ArchivePath && $CurrentTestName);
+    archive_files($CurrentTestName, $ServRoot);
```

Comment:
Better swap the order of the function arguments. I think it's more natural to put the source directory before the destination.